### PR TITLE
Generate semantic similarity subsumers

### PR DESCRIPTION
**Pull request by Shalsh23**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -87,6 +87,7 @@ $(BUILD_DIR)/query-subsumers.ofn: $(BUILD_DIR)/qualities.txt $(BUILD_DIR)/anatom
 
 # Create Similarity-Subsumers
 $(BUILD_DIR)/similarity-subsumers.ofn: $(BUILD_DIR)/qualities.txt $(BUILD_DIR)/anatomical_entities.txt
```
Comment (Shalsh23) [inline Makefile]: Why does this need to depend on `qualities`? It doesn't seem to use it in the code in `PhenoscapeKB.scala`.

Comment (Shalsh23) [reply Makefile]: Also, what will go in the target for `query subsumers`? I couldn't find anything relevant in `PhenoscapeKB.scala`.

Comment (balhoff) [reply Makefile]: I doesn't need qualities at present. I used to use some patterns that involved qualities in the past; maybe we will add that back later.

Comment (balhoff) [reply Makefile]: I think 'query subsumers' is all the patterns you've been adding. If that doesn't seem right, we can talk about it when we meet.

**Pull request merged**